src/cumin/pagination.clj

```diff
@@ -22,13 +22,14 @@
     (dec curr-page)))
 
 (defn- select-total [query]
-  (-> query
-      (dissoc ::pagination :post-queries :limit :offset :order)
-      (assoc :fields [])
-      (aggregate (count :id) :count)
-      (select)
-      (ffirst)
-      (last)))
+  (let [pk (get-in query [:ent :pk] :id)]
```

Comment:
Good question. I could've sworn I'd seen MySQL performing way slower w/ `count(*)` than w/ `count(id)` (years ago), but everything I'm seeing now says otherwise (and I don't use MySQL anymore anyway). Will update; that'll be more straightforward.